src/NServiceBus.Testing.Fakes/TestableAuditContext.cs

```diff
@@ -11,9 +12,13 @@
     public partial class TestableAuditContext : TestableBehaviorContext, IAuditContext
     {
         /// <summary>
-        /// Contains the information added by <see cref="AddedAuditData" />.
+        /// Contains the information added by <see cref="AddAuditData" />.
         /// </summary>
-        public IDictionary<string, string> AddedAuditData { get; } = new Dictionary<string, string>();
+        [ObsoleteEx(
```

Comment:
Not sure if this works? NServiceBus.Testing consumes this file and does use Obsolete.Fody but what if NServiceBus.Testing is on a different version than core?

Reply:
I would not bother to obsolete here. Seems YAGNI. The production code will already get obsoletion warnings.